CODEOWNERS

```diff
@@ -30,7 +30,7 @@
 
 /Workbooks/Azure*Blockchain/** @keikumata @microsoft/applicationinsights-devtools
 
-/Workbooks/HDInsight/** @tylerfox @microsoft/applicationinsights-devtools
+/Workbooks/HDInsight/** @blc16 @hdinsight-workbooks-team @microsoft/applicationinsights-devtools
```

Comment:
this is intended to be the literal team, right?  `@microsoft/hdinsight-workbooks-team` is a valid team name, this is just a github user with that name, which is not valid?  and if you @blc16 are already in that team, you don't need to be mentioned explicitly here.

Reply:
fixed this